src/popup/components/MultisigVaultCreationProgress.vue

```diff
@@ -3,30 +3,33 @@
     <h2 class="title">
       {{ $t('modals.creatingMultisigAccount.title') }}
     </h2>
-    <ProgressBar :progress="creatingVaultProgress" />
+    <ProgressBar :progress="progressPercentage" />
     <div class="steps-list">
       <div
-        v-for="step in steps"
-        :key="step.id"
+        v-for="(step, index) in steps"
+        :key="`step${index}`"
         class="step-item"
         :class="{
-          active: currentProgressStepId >= step.id,
-          completed: currentProgressStepId > step.id,
+          completed: isStepCompleted(index) || isStepCurrent(index),
         }"
       >
+        <PendingIcon
+          v-if="isStepCurrent(index)"
+          class="step-pending-icon"
+        />
         <CheckSuccessCircleIcon
-          v-if="currentProgressStepId > step.id || !!multisigAccount"
+          v-else-if="isStepCompleted(index)"
           class="step-success-icon"
         />
         <div
           v-else
-          class="step-item-id"
+          class="step-number"
         >
-          {{ step.id }}
+          {{ index + 1 }}
         </div>
 
         <div class="step-item-name">
-          {{ getStepText(step) }}
+          {{ step.text }}<span v-if="isStepCurrent(index)">...</span>
```

Comment:
"..." ->  Hmm i think good option is create constant for this

Reply:
I'm gonna use `&hellip;` because using JS const is a bad idea in my opinion.